config/sitemap.rb

```diff
@@ -1,6 +1,8 @@
 require 'sitemap_generator'
 
-SitemapGenerator::Sitemap.default_host = "#{Settings.host}#{Settings.base_path}"
```

Comment:
Is this a `TODO` we should.. uh.. do?

Reply:
Ah, I think this was caused by the generator that was run, I'll submit a follow on PR to address if that's 🆒

Reply:
Added in #27